src/main/java/core/WebDriverSingleton.java

```diff
@@ -1,5 +1,6 @@
 package core;
 
+import logging.CustomLogger;
```

Comment:
Change WebDriver to Custom

public class WebDriverSingleton {

    private static CustomWebDriver instance;
    private static RunLocationCreator runLocationCreator;

    private WebDriverSingleton() {
    }

    public static void selectDriverType(RunLocationCreator locationCreator) {
        runLocationCreator = locationCreator;
    }

    public static CustomWebDriver getWebDriverInstance() {
        if (instance != null) {
            return instance;
        }
        return instance = init();
    }


    private static CustomWebDriver init() {
        if (null == runLocationCreator) {
            runLocationCreator = new LocalCreator();
        }
        WebDriver driver = runLocationCreator.getWebDriver();
        driver.manage().timeouts().pageLoadTimeout(20, TimeUnit.SECONDS);
        driver.manage().timeouts().implicitlyWait(20, TimeUnit.SECONDS);
        driver.manage().window().maximize();
        return new CustomWebDriver(driver);
    }